### ISSUE TITLE
Missing accessibility of attributes and methods in 1st class diagram for DG

### ISSUE THREAD

**Issue by nshian** [nshian/pe#17]
The accessibility of the attributes and methods in the classes for the 1st class diagram in the DG should have been included e.g. for a private attribute, prefix it with a "-" or for a public method, prefix it with a "+".

These details are missing from the class diagram which can cause confusion for the programmer reviewing the guide.

![image.png](https://raw.githubusercontent.com/nshian/pe/main/files/1cd233f8-3444-40c1-b5ac-f9fff4ed7265.png)

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
No details provided by team.

# Items for the Tester to Verify

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.High`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------
